### DISCUSSION TITLE
Bump fflate version

### DISCUSSION THREAD

**Discussion by 101arrowz** [protectwise/troika#99] [General]
Hello! I noticed you added `fflate` in 2ae29faffcec2302453ce9dabac633ade8181127 but had some issues with importing properly. I wanted to let you know that I fixed the problem with an extraneous `require` appearing, you can update to v0.5.0 on NPM to make it work properly. I hope the library works well for you :)

**Comment by lojjic** [comment]
Great, thanks for the heads up! I'll give it a shot.

Really great job on `fflate`, by the way. I'm very impressed. :)
